res/openvpn-gui-res-zh-hans.rc

```diff
@@ -285,7 +285,7 @@ CAPTION "Select Certificate"
 FONT 8, "Segoe UI"
 LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 BEGIN
-    LTEXT "PKCS11 Certificates available:", ID_TXT_PKCS11, 17, 12, 171, 12
+    LTEXT "PKCS11证书可用：", ID_TXT_PKCS11, 17, 12, 171, 12
     CONTROL "", ID_LVW_PKCS11, "SysListView32", LVS_REPORT | LVS_SINGLESEL | WS_BORDER | WS_TABSTOP, 17, 25, 171,150
     PUSHBUTTON "&OK", IDOK, 20, 200, 50, 14, BS_DEFPUSHBUTTON | WS_TABSTOP
     PUSHBUTTON "&Cancel", IDCANCEL, 90, 200, 50, 14, BS_PUSHBUTTON | WS_TABSTOP
```

Comment:
OK, Cancel and Retry (next line) do not need translation? In some dialogs above I see OK and Cancel translated. 
If you make changes just push a commit on top of this -- we can squash while merging.

Reply:
OK